luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/DefaultLogicalTableGroupFactory.java

```diff
@@ -0,0 +1,146 @@
+    private static final String ENTITY_TYPE = "default logical table group";
+    private static final String GRANULARITY_PARSING_ERROR =
+            "granularity: %s does not exist in the granularityDictionary";
+    private static final String GRANULARITY_DICTIONARY_MISSING = "granularityDictionary missing from the " +
+            "LuthierIndustrialPark. Will not build " + ENTITY_TYPE + ": %s.";
+
+    /**
+     * Build a group of LogicalTable, it has one LogicalTable per granularity.
+     *
+     * @param name  apiName of the LogicalTable
+     * @param configTable  the LogicalTableGroup's configuration
+     * @param resourceFactories  the source for locating dependent objects
+     *
+     * @return  A newly constructed config instance for the name and config provided
+     */
+    @Override
+    public LogicalTableGroup build(String name, ObjectNode configTable, LuthierIndustrialPark resourceFactories) {
+        LogicalTableGroup logicalTableGroup = new LogicalTableGroup();
+        GranularityDictionary granularityDictionary = resourceFactories.getGranularityDictionary();
+        if (granularityDictionary == null) {
+            LOG.warn(String.format(GRANULARITY_DICTIONARY_MISSING, name));
+            return logicalTableGroup;
+        }
+        validateFields(name, configTable);
+
+        String category = configTable.get("category").textValue();
+        String longName = configTable.get("longName").textValue();
+        ReadablePeriod retention = Period.parse(configTable.get("retention").textValue());
+        String description = configTable.get("description").textValue();
+
+        MetricDictionary metricDictionary = resourceFactories.getMetricDictionary();
+        LinkedHashSet<Dimension> dimensions = new LinkedHashSet<>();
+        LinkedHashSet<PhysicalTable> physicalTables = new LinkedHashSet<>();
+        LinkedHashSet<ApiMetricName> apiMetricNames = new LinkedHashSet<>();
+        configTable.get("dimensions").forEach(
+                node -> dimensions.add(resourceFactories.getDimension(node.textValue()))
+        );
+        configTable.get("physicalTables").forEach(
+                node -> physicalTables.add(resourceFactories.getPhysicalTable(node.textValue()))
+        );
+        configTable.get("metrics").forEach(
+                node -> {
+                    /* go to the LIP to retrieve the metric */
+                    /* Must be done after having actual metric builders! */
+                    // apiMetricNames.add(new LuthierApiMetricName(node.textValue(), granularity));
+                    // metricDictionary.add();
+                }
+        );
+        TableGroup tableGroup = new TableGroup(
+                physicalTables,
+                apiMetricNames,
+                dimensions
+        );
+
+        configTable.get("granularities").forEach(
+                 node -> {
+                    GranularityParser granularityParser = new StandardGranularityParser(
+                    );
```

Comment:
This doesn't need to be wrapped. Also, we should just lift this up to a field and build it when the factory is constructed.
No sense in building a new one over and over again.

Reply:
done